### PR TITLE
🐛 Room show/hide on map if enabled/disabled in admin panel

### PR DESCRIPTION
**Pull request by IvanBuljovcic**
Show/Hide room on the partymap/themecamp if it is Enabled/Disabled in the Admin panel

### CODE REVIEW

File: src/components/templates/Camp/components/MapRoomOverlay.tsx [hunk 1]
```diff
@@ -139,6 +139,8 @@ export const MapRoomOverlay: React.FC<MapRoomOverlayProps> = ({
     (attendances[`${venue.name}/${room.title}`] ?? 0) +
     (room.attendanceBoost ?? 0);
 
+  if (!room.isEnabled) return <></>;
```
Comment (0xdevalias) [inline src/components/templates/Camp/components/MapRoomOverlay.tsx:142]: Fragments are only really meant for grouping other sibling JSX elements together when we don't want to emit a `<div>`/similar to give them a common parent. In this case, we would want to return `null`

- https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering

**Review by 0xdevalias — CHANGES_REQUESTED** [review]
Can you please update the `<></>` returns to `null` as per in-lined comment, then this should be good to go!

**Comment by 0xdevalias** [comment]
[Room turning on and off not working on partymap and themecamp templates](https://trello.com/c/689NmRaI/354-room-turning-on-and-off-not-working-on-partymap-and-themecamp-templates)

**Review by 0xdevalias — APPROVED** [review]
LGTM!

**Comment by thecadams** [comment]
Great to see this merged. Quickly wondering, what about the third stage of a room when it’s disabled. Should we have it be at say 50% opacity and not clickable?

**Pull request merged**